app/src/main/java/com/example/cooky/ui/recipe/RecipeFragment.kt

```diff
@@ -174,5 +203,9 @@ class RecipeFragment : BaseFragment<FragmentRecipeBinding, RecipeViewModel>() {
         private const val ROTATE_DURATION = 100L
         private const val ROTATE_INVERSE = 180f
         private const val ROTATE_DEFAULT = 0f
+        private const val MAX_RECENTLY_RECIPES = 8
+        private const val INDEX_FIRST = 0
+        const val RECENTLY_ID = "recently id"
+        const val NULL = "null"
```

Comment:
cho vào constant đi anh 
```suggestion
        const val STRING_NULL = "null"
```